<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BruteForceDialog</class>
 <widget class="QMainWindow" name="BruteForceDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1013</width>
    <height>428</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Brute Force Attack</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="bfan_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>811</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Now, the application wish to conduct a brute force attack against your selected IP camera</string>
    </property>
   </widget>
   <widget class="QLabel" name="bfan_label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>811</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>However, the brute force attack may be taken hours to be completed</string>
    </property>
   </widget>
   <widget class="QLabel" name="bfan_label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>871</width>
      <height>41</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>120</red>
          <green>120</green>
          <blue>120</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>(As each attempt is taken 30 seconds to be completed to test the connection to your IP Camera)</string>
    </property>
   </widget>
   <widget class="QLabel" name="bfan_label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>991</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>And this attack will give you an overview if the credential you're using for your IP Camera is secure or insecure</string>
    </property>
   </widget>
   <widget class="QLabel" name="bfan_label_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>180</y>
      <width>991</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>If you choose to proceed with the brute force attack, here are some insturctions:</string>
    </property>
   </widget>
   <widget class="QLabel" name="bfan_label_6">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>210</y>
      <width>991</width>
      <height>41</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>170</red>
          <green>0</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>170</red>
          <green>0</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>120</red>
          <green>120</green>
          <blue>120</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>1) Make sure your local network connection is stable</string>
    </property>
   </widget>
   <widget class="QLabel" name="bfan_label_7">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>240</y>
      <width>991</width>
      <height>41</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>170</red>
          <green>0</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>170</red>
          <green>0</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>120</red>
          <green>120</green>
          <blue>120</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>2) You can press 'q' to quit the RTSP streaming at any time</string>
    </property>
   </widget>
   <widget class="QLabel" name="bfan_label_8">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>270</y>
      <width>991</width>
      <height>41</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>170</red>
          <green>0</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>170</red>
          <green>0</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>120</red>
          <green>120</green>
          <blue>120</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>3) You can press 's' to take screenshot of your IP Camera footage</string>
    </property>
   </widget>
   <widget class="QPushButton" name="proceed_pushButton">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>320</y>
      <width>311</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Proceed</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cancel_pushButton">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>320</y>
      <width>311</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Cancel</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1013</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
